shared_model/cryptography/crypto_provider/abstract_crypto_model_signer.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_ABSTRACT_CRYPTO_MODEL_SIGNER_HPP
+#define IROHA_ABSTRACT_CRYPTO_MODEL_SIGNER_HPP
+
+namespace shared_model {
+  namespace crypto {
+
+    // signer which can be overloaded for dynamic polymorphism
```

Comment:
Seems that this comment is redundant and there is a lack of doc string. I suggest to leave it here.